plugins/baseplugin/src/main/java/org/apache/unomi/plugins/baseplugin/conditions/PropertyConditionEvaluator.java

```diff
@@ -88,9 +88,9 @@ private int compare(Object actualValue, String expectedValue, Object expectedVal
         }
     }
 
-    private boolean compareMultivalue(Object actualValue, List<?> expectedValues, List<?> expectedValuesDate, List<?> expectedValuesNumber, List<?> expectedValuesDateExpr, String op) {
+    private boolean compareValues(Object actualValue, List<?> expectedValues, List<?> expectedValuesInteger, List<?> expectedValuesDouble, List<?> expectedValuesDate, List<?> expectedValuesDateExpr, String op) {
         @SuppressWarnings("unchecked")
-        List<?> expected = ObjectUtils.firstNonNull(expectedValues, expectedValuesDate, expectedValuesNumber);
+        List<?> expected = ObjectUtils.firstNonNull(expectedValues, expectedValuesDate, expectedValuesInteger, expectedValuesDouble);
```

Comment:
Seem's that expectedValuesDateExpr are not used here. 
either we should remove this parameter to avoid confusion or handle it by transforming this date expressions to date and compare them like regular expectedValuesDate

Reply:
agree, implemented! Thank you